docs/src/GettingStarted.md

```diff
@@ -0,0 +1,108 @@
+
+
+## Create your repository
+Start by creating a new private repository on GitHub based on the [lf-3pi-template](https://github.com/lf-lang/lf-3pi-template) repository, which provides a starting point for students to carry out the exercises in this lab and to develop further applications using the [Raspberry Pi Pico board](https://www.raspberrypi.com/products/raspberry-pi-pico/) and the [Pololu 3pi+ 2040 robot](https://www.pololu.com/docs/0J86).
+
+Navigate to the [lf-3pi-template](https://github.com/lf-lang/lf-3pi-template) repository.  Select "Use this template" and "Create a new repository", as shown here:
+
+<img src="img/3piTemplate.png" alt="Template repo"/>
+
+Give your repo a name and click on "Create repository":
+
+<img src="img/my-3pi.png" alt="new repo"/>
+
+## Clone your repository
+On the command line on your host machine, clone the repo.
+For an SSH-based setup, run:
+
+```bash
+$ git clone git@github.com/<username>/<reponame>.git
+```
+
+Or, for a setup that uses GitHub CLI, run:
+
+```bash
+$ gh repo clone <username>/<reponame>
+```
+
+The template includes [raspberrypi/pico-sdk](https://github.com/raspberrypi/pico-sdk) as a submodule, which itself also has a lot of submodules. We recommend against using the `--recursive` flag because we do not need to recursively clone the submodules inside of `pico-sdk`. Instead, change directory into the root of your clone and run:
+
+```bash
+$ git submodule update --init
+```
+
+If  `pico-sdk` was checked out correctly, `git submodule` will show the hash _without_ a `-` preceding it,
+e.g.: `6a7db34ff63345a7badec79ebea3aaef1712f374 pico-sdk (1.5.1)`.
+
+
+## Configure Nix
+
+To create a reproducible unix shell environment that installs all required dependency applications, we use the [nix](https://nixos.org) package manager, which has support for Linux, macOS, and Windows (via WSL). See [prerequisites](Prerequisites.html) for installation instructions. If you prefer to manage dependencies yourself and not rely on `nix`, follow the [instructions for a non-`nix` setup](Non-Nix.html).
+
+After installation, run the following in the shell to enable the experimental nix flakes feature, which helps to create more consistent builds:
+
+```bash
+$ mkdir -p ~/.config/nix
+$ echo "experimental-features = nix-command flakes" >> ~/.config/nix/nix.conf
+```
+
+To install the dependencies, run the following in the root of your repository:
+
+```bash
+$ nix develop
+```
+
+This should automatically download and install specific revisions of the `gcc-arm` toolchain, `openocd`, and `picotool`. These packages will be required compiling, flashing and debugging C code for the RP2040.
+(You can alternatively manually [install the Raspberry Pi Pico Tools](InstallingPicoTools.md).)
```

Comment:
This link is a 404, probably because InstallingPicoTools.md is not in the SUMMARY.md file.

Reply:
Fixed.